lib/kafo/kafo_configure.rb

```diff
@@ -40,11 +41,17 @@ def hooking
     def initialize(*args)
       self.class.logger           = Logger.new
       self.class.exit_handler     = ExitHandler.new
+      self.class.scenario_manager = ScenarioManager.new(CONFIG_DIR || CONFIG_FILE)
```

Comment:
`CONFIG_DIR` might not be defined at this time (old bin script), if it's defined but set to nil/false it would crash on undefined `CONFIG_FILE` on new bin script I think